src/formatDate.js

```diff
@@ -8,7 +8,47 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const split = fromFormat[3];
+  const dayOfYear = date.split(split);
```

Comment:
The variable name 'dayOfYear' is misleading as it suggests a day within a year, not parts of the date. Consider renaming it to something more accurate like 'dateParts'.